### PR TITLE
rebranded accordion

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

## Summary

Rebrand of Accordion Component

## What I did

- Updated tokens to match Hopper tokens.

### CODE REVIEW

File: packages/components/src/accordion/src/Accordion.css [hunk 1]
```diff
@@ -41,27 +42,27 @@
...
 }
 
 /* TRIGGER | BORDERED */
 .o-ui-accordion-bordered .o-ui-accordion-trigger {
-    background-color: var(--o-ui-bg-alias-surface);
-    padding: calc(var(--o-ui-sp-4) - 1px) var(--o-ui-sp-5) var(--o-ui-sp-4) var(--o-ui-sp-6);
-    min-height: var(--o-ui-sz-8);
-    border-top: 1px solid var(--o-ui-b-alias-low-break);
+    background-color: var(--hop-neutral-surface);
+    padding: calc(var(--hop-space-inset-md) - 1px) var(--hop-space-inset-lg) var(--hop-space-inset-md) var(--hop-space-inset-lg);
```
Comment (vicky-comeau) [inline packages/components/src/accordion/src/Accordion.css:57]: What's the point of the -1? Do we still need it?

File: packages/components/src/accordion/src/Accordion.css [hunk 2]
```diff
@@ -41,27 +42,27 @@
...
+    min-height: 3.5rem;
+    border-top: 0.0625rem solid var(--hop-neutral-border-weakest);
 }
 
 /* TRIGGER | BORDERED | HOVER */
 .o-ui-accordion .o-ui-accordion-trigger:not([disabled]):hover,
 .o-ui-accordion .o-ui-accordion-trigger-hover:not([disabled]) {
-    background-color: var(--o-ui-bg-alias-mid-break);
+    background-color: var(--hop-neutral-surface-weak);
```
Comment (vicky-comeau) [inline packages/components/src/accordion/src/Accordion.css:65]: I thought you mentioned following styles for disclosure. If so, the hover background color should be: --hop-neutral-surface-hover

File: packages/components/src/accordion/src/Accordion.css (hunk 1 above)
Comment (fraincs) [reply packages/components/src/accordion/src/Accordion.css:57]: Yep it's linked to the border when selected.

File: packages/components/src/accordion/src/Accordion.css (hunk 2 above)
Comment (fraincs) [reply packages/components/src/accordion/src/Accordion.css:65]: Quick notice : as there is no Accordion in Hopper, and Disclosure looks too much like an accordion in it's current state, the goal was to reskin Orbit accordion with Hopper values.

Although it makes sense to use the hover color in this situation I'll fix it.

File: packages/components/src/accordion/src/Accordion.css [hunk 3]
```diff
@@ -41,27 +42,27 @@
...
     font-weight: inherit;
     position: relative;
-    padding: 0 var(--o-ui-sp-2);
+    padding: 0 var(--hop-space-inset-sm);
     font-family: inherit;
     font-size: 100%;
 }
 
 /* TRIGGER | BORDERED */
 .o-ui-accordion-bordered .o-ui-accordion-trigger {
```
Comment (vicky-comeau) [inline packages/components/src/accordion/src/Accordion.css:55]: not sure if these follow the inline Disclosure, if so, you may need a border: 
image
I realize there is a borderless and a bordered version, but the bordered is missing a border when opened.

**Review by vicky-comeau — APPROVED** [review]

**Pull request merged**